docs/build-process/schema-descriptor-build-process.md

```diff
@@ -0,0 +1,19 @@
+# Building a schema descriptor in the FPL
+This page explains how to build a schema descriptor in a SpatialOS project that’s using the flexible project layout (FPL).
```

Comment:
@anne-edwards Fyi: There is no glossary entry for the schema descriptor at the moment. :-/

Reply:
Thanks - I've just looked in the main docs and there's really nothing at all about schema descriptors... Is this something we should add to the main glossary? It looks like we could just lift the first paragraph and the note below and add it to the glossary.

Reply:
@JonasImprobable do you think I should just add this to the main glossary, using your wording?